pkg/Microsoft.ML.CodeGen/Microsoft.ML.CodeGen.nupkgproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
```

Comment:
Name this file the exact same name as the `.csproj`, only change the extension from `.csproj` to `.nupkgproj`.

so `Microsoft.ML.CodeGenerator.nupkgproj`. The folder should be named the same too.